### DISCUSSION TITLE
3D Berkovich Indentation Example Units

### DISCUSSION THREAD

**Discussion by Vandenbg** [idaholab/moose#21535] [Q&A General] [answered]
Hello,

I am having trouble establishing the units used for the example in:

moose/modules/contact/examples/3d_berkovich/

The journal file used to create the mesh does not specify units and neither does the input file itself. The mesh displaces at the same rate as time ending at 1.8. Units in finite element analysis are arbitrary, however, given the examples inputs there must be units associated with the displacement and time. I am assuming seconds and micrometers. Is there something I am missing, is there any way to confirm this?

**Comment by GiudGiud** [comment, marked as answer]
Hello

There is documentation for this example
https://mooseframework.inl.gov/modules/contact/BerkovichIndenterNodeFace.html
does it help with the units? Seems time is in seconds.

Guillaume

**Reply by Vandenbg** [reply]
Thank you for the quick reply, this does confirm time is in seconds. I was not able to gleam other units from this documentation or the nearby 2d examples when searching earlier. 

edit: it does note that the parameters for the material are arbitrary, however, I don't think this means that the length scale is too.

**Reply by GiudGiud** [reply]
I think you should be able to determine whether it's in m, mm or micrometer from your knowledge of the problem and the material properties a this point.

**Reply by Vandenbg** [reply]
Given the stress output by running the simulation, it would make sense that the units are micrometers. Thank you for the help.

**Reply by jiangwen84** [reply]
If I recall correctly, the length unit is micro meter and the stress is mpa.
